manifests/server/config.pp

```diff
@@ -147,6 +147,17 @@
                   Exec['puppet_server_config-create_ssl_dir'],
                   ],
     }
+  } elsif $::puppet::server::ca_crl_sync {
+    # If not a ca AND sync the crl from the ca master
+    if $::server_facts['servername'] != undef {
```

Comment:
I will conceed that defined('$::var') will work but as theforeman/puppet-puppet is used for more than just foreman installs relaying on a topscope which is only set with foreman install limits the efficacy of this fix. We need to identify a better means of determining agent run vs apply run. I am open to suggestions